lib/api/_loaders/element-global.js

```diff
@@ -89,7 +90,23 @@ class ElementGlobal {
 
     const byLocator = Locator.create(locator);
 
-    this.element = await this.transport.driver.wait(until.elementLocated(byLocator), this.timeout, null, this.retryInterval);
+    this.setPropertiesFromElement(locator);
+
+    const elements = await this.transport.driver.wait(until.elementsLocated(byLocator), this.timeout, null, this.retryInterval);
+
+    if (elements.length === 0) {
+      throw new Error(`Element ${byLocator} was not found.`);
```

Comment:
Can you modify this error message as well, like you modified the error message for index out of bounds? Refer the same file I shared before.